.gitignore

```diff
@@ -0,0 +1,5 @@
+# Python cache files
```

Comment:
Yeah, I'm still thinking those are meh - but I think there is no other way if multiple folks will edit the repo...